scripts/test_custom_querier.sh

```diff
@@ -4,11 +4,12 @@ CHAINID2=test-2
 NEUTRON_DIR=${NEUTRON_DIR:-../neutron}
 HOME=${NEUTRON_DIR}/data/test-1/
 HOME2=${NEUTRON_DIR}/data/test-2/
-KEY=demowallet1
+KEY=demowallet3
```

Comment:
in the `test_crud` and `ibc_transfer` scripts `demowallet3` is the faucet's account. why change it to the main actor here? this is confusing compared to the scenario where we have same roles for wallets across all tests

Reply:
You are right I need to modify code to use faucet address everywhere.